### DISCUSSION TITLE
[FEATURE REQUEST] Switch to /etc/crontab for cron states

### DISCUSSION THREAD

**Discussion by udf2457** [saltstack/salt#67516] [Feature Requests]
At the moment `cron.present` places the entry in the root user's cron (i.e. the same as would happen if you ran `crontab -u root -e`).

The biggest problem with this approach instead of `/etc/crontab` is that you cannot assign a non-root user to the job, which you can do with `/etc/crontab`.  Sure you could prefix the `cron.present` entry with `sudo` but that's not exactly best practice (and is a bit messy).

As per the comments in the header of `/etc/crontab`;

> Unlike any other crontab you don't have to run the `crontab'
> command to install the new version when you edit this file
> and files in /etc/cron.d. These files also have username fields,
> that none of the other crontabs do.

**Comment by mihai-stancu** [comment]
Why not `/etc/cron.d/name_of_identifier` ?
